src/codemagic/tools/firebase/firebase.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+
+import argparse
+import json
+
+from codemagic import cli
+from codemagic.firebase.api_client import FirebaseApiClient
+
+from .action_groups import ReleasesActionGroup
+from .actions import GetLatestBuildVersionAction
+from .arguments import FirebaseArgument
+
+
+@cli.common_arguments(FirebaseArgument.FIREBASE_SERVICE_ACCOUNT_CREDENTIALS)
```

Comment:
Given that all REST resources in [Firebase App Distribution API](https://firebase.google.com/docs/reference/app-distribution/rest) require project ID then I find that `ReleasesArgument.PROJECT_ID` could be better suited as a common argument instead of specifying it on all individual actions separately.